### PR TITLE
Add wait for iptables

### PR DESCRIPTION
**Pull request by mariusSincovici**
In some cases iptables are locked and the command fails with `Another app is currently holding the xtables lock. Perhaps you want to use the -w option?`. To prevent this add `-w` into the param list for `iptables` Add also for how long to wait for the lock to be free, otherwise the daemon gets stuck.

### CODE REVIEW

**Comment by piotrjurkiewicz** [comment]
There is also an iptables call in `daemon/firewall/iptables/iptables_test.go:676`

**Review by piotrjurkiewicz — APPROVED** [review]

**Review by devzbysiu — APPROVED** [review]

File: internal/constants.go [hunk 1]
```diff
@@ -82,6 +82,8 @@ const (
 	FileshareLogFileName = "nordfileshare" + LogFileExtension
 
 	LogFileExtension = ".log"
+
+	SecondsToWaitForIptablesLock = "5" // use string to easier concatenate into the command args
```
Comment (keliramu) [inline internal/constants.go:86]: IMO, this parameters should live in `firewall` top package

File: daemon/firewall/iptables/iptables.go [hunk 2]
```diff
@@ -96,7 +96,7 @@ func (ipt *IPTables) applyRule(rule firewall.Rule, add bool) error {
 		}
 		for _, ipTableRule := range ipTablesRules {
 			// -w does not accept arguments on older iptables versions
-			args := fmt.Sprintf("%s %s -w", flag, ipTableRule)
+			args := fmt.Sprintf("%s %s -w "+internal.SecondsToWaitForIptablesLock, flag, ipTableRule)
```
Comment (keliramu) [inline daemon/firewall/iptables/iptables.go:99]: Let's be consistent: `-w` parameter should be passed via `%d` same
```
fmt.Sprintf("%s %s -w %d", internal.SecondsToWaitForIptablesLock, flag, ipTableRule)
```

**Review by keliramu — APPROVED** [review]

File: internal/constants.go (hunk 1 above)
Comment (keliramu) [reply internal/constants.go:86]: It should be integer, I guess

File: daemon/firewall/iptables/iptables.go (hunk 2 above)
Comment (keliramu) [reply daemon/firewall/iptables/iptables.go:99]: Or there is some reason you are doing this way?

Comment (keliramu) [reply daemon/firewall/iptables/iptables.go:99]: Sorry, I see your comment one line above, ok

**Pull request merged**